core/src/test/scala/org/apache/spark/h2o/utils/SharedSparkTestContext.scala

```diff
@@ -51,26 +52,51 @@ trait SharedSparkTestContext extends SparkTestContext { self: Suite =>
     super.afterAll()
   }
 
-  def buildChunks[T: ClassTag](fname: String, data: Array[T], cidx: Integer, h2oType: Array[Byte]): Chunk = {
+  def makeH2OFrame[T: ClassTag](fname: String, colNames: Array[String], chunkLayout: Array[Long],
```

Comment:
I think JIRA would be better, it's easier to track than to track TODOs in the code. It's also more generic JIRA - replace all code which we use to create small frames with this builder.